src/scss/components/optional/_apps-banner.scss

```diff
@@ -1,15 +1,16 @@
-/**
- * Apps Banner Component
- * =================================
- * Used to display app promo banner on the page.
- *
- * *The `c-appBaner` component is an optional mixin within Fozzie — if you'd like to use it in your project you can include it by adding `@include appsBanner();` into your SCSS dependencies file.*
- *
- * Documentation:
- * https://fozzie.just-eat.com/styleguide/ui-components/apps-banner
- */
-
 @mixin appsBanner() {
```

Comment:
Now that all components are going to be optional do we still need the`/optional` directory?

Reply:
I'll be removing that in my next PR – didn't want to re-arrange it in this one as it kind of white-washes over the changes in files then in Github.

Reply:
Makes sense.